src/monodroid/jni/Android.mk

```diff
@@ -3,9 +3,9 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
 ifeq ($(CONFIGURATION),Debug)
-COMMON_CFLAGS ?= -ggdb3 -O0 -fno-omit-frame-pointer
+COMMON_CFLAGS ?= -ggdb3 -O0 -fno-omit-frame-pointer -DDEBUG=1
 else
-COMMON_CFLAGS ?= -g -O2 -DRELEASE=1
```

Comment:
I don't understand why it's better to have `-DRELEASE=1` within `jni\Application.mk` as opposed to within `jni/Android.mk`.